core/settings/_breakpoints.scss

```diff
@@ -59,7 +59,7 @@ $breakpoints: (
 
 /**
  * A global setting to define the common breakpoint(s) for Scally and your
- * project. This is mainly used by the Generate at breakpoints mixin.
+ * project. This is mainly used by the `generate-at-breakpoints()` mixin.
```

Comment:
Lol. I didn't realise there was a convention. You've just said before to always wrap code in `` so I did :) Sorry for breakibg you OCD :D